src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -92,7 +106,13 @@ export class CalciteDropdownItem {
           role="menuitem"
           aria-selected={this.active.toString()}
         >
+          {this.icon && this.iconPosition === "start" ? (
```

Comment:
suggestion: maybe have a const for the node `<calcite-icon icon={this.icon} scale={iconScale}></calcite-icon>` since it is repeated.

Another option instead of moving the node would be to have the host container display flex and row and then when "end" it could display row-reverse.

Last option would be to have the nodes in array that you reverse. `[iconNode, <slot>]`

Reply:
Yeah good call, will update.